docs/docs/recipes/manage-users/README.md

```diff
@@ -3,3 +3,63 @@ sidebar_position: 6
 ---
 
 # 🧑‍🚀 Manage users
+
+## You don't need a user table in the database
+
+Before taking Logto as your identity solution, you may think about how to organize users and related info.
+Traditionally, developers create a user table in a database containing the user info, then create a many-to-one relation in other tables, with a `user_id` column pointing to the user table.
+
+Taking Google as an example for other identity providers, you may create a column named `google_id` to associate the users who "Sign in with Google".
+For Logto, you may also intend to create a column `logto_id`, but it is unnecessary.
```

Comment:
it doesn't like a proper example.

- i didn't see the connection between google and logto
- didn't understand what's logto_id for